documentation/modules/configuring/con-config-mirrormaker2-high-volume-messages.adoc

```diff
@@ -6,26 +6,54 @@
 = Handling high volumes of messages
 
 [role="_abstract"]
-If your MirrorMaker 2.0 deployment is going to be handling a high volume of messages, you might need to adjust its configuration to support it.
+If your MirrorMaker 2.0 deployment is going to be handling a high volume of messages, you might need to adjust its configuration to support it. 
+The configuration can help manage throughput and issues with latency. 
```

Comment:
Could we merge this sentence with the one above? It looks a bit strange bit itself

Reply:
:+1: 
Works better with the new examples too